### PR TITLE
DOCSP-27964-release-notes-1.8.0

### PR DESCRIPTION
**Pull request by davemungo**
## DESCRIPTION

[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-27964-release-notes-1.8.0/changelog/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-27964)

[build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6400cfb405dd3cc8141adfdd)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jason-price-mongodb — APPROVED** [review]
@davemungo LGTM. Cheers.

File: source/changelog.txt [hunk 1]
```diff
@@ -12,6 +12,43 @@ Release Notes
...
+  inherited enumerable keys to the server.
+
+Issues fixed
+~~~~~~~~~~~~
+
+- :issue:`MONGOSH-1358` Updates for the 5.1.0 :driver:`Node.js driver
+  </node/>`. 
+- :issue:`MONGOSH-1336` Performance improvement some use cases including
+  the ``--version`` and ``--build-info`` flags.
+- :issue:`MONGOSH-1316` Surfaces the ``createEncryptedCollection``
```
Comment (himanshusinghs) [inline source/changelog.txt:46]: I am assuming that an `issue` means a bug, following which this one is not really an `issue` but more like a small feature. Please let me know if I am interpreting it correctly.

Comment (davemungo) [reply source/changelog.txt:46]: Not really, that's the heading we put at the top of the list on [the server pages](https://www.mongodb.com/docs/manual/release-notes/6.0/). Those pages also have section headers (like "Compatibility Changes" here.) 

mongosh releases don't follow the same process though. I've changed the header to make it more general.

**Review by davemungo — COMMENTED** [review]
@himanshusinghs thanks for your feedback. This is ready for another look

File: source/changelog.txt (hunk 1 above)
Comment (himanshusinghs) [reply source/changelog.txt:46]: Thanks @davemungo

**Review by himanshusinghs — APPROVED** [review]

**Pull request merged**